R/drive_update.R

```diff
@@ -6,7 +6,8 @@
 #'
 #' @template file
 #' @template media
-#' @param ... Parameters to pass along to the API query. NOT IMPLEMENTED YET.
+#' @param ... Parameters to pass along to the [API query](https://developers.google.com/drive/v3/reference/files/update).
+#'   These parameters can be used to update metadata.
```

Comment:
This is what I want for `drive_cp()` documentation of `...`. In fact, could they share a template? You can move the endpoint-specific hyperlink into another part of the help file.